src/proto/shim/mod.rs

```diff
@@ -4,6 +4,37 @@ use crate::proto::Protocol;
 use crate::result::Error;
 use crate::{unsafe_guid, Result, Status};
 use core::convert::TryInto;
+use core::ffi::c_void;
+use core::mem::MaybeUninit;
+
+// The PE_COFF_LOADER_IMAGE_CONTEXT type. None of our methods need to inspect
```

Comment:
Just a minor style issue, could you write the `PE_COFF_LOADER_IMAGE_CONTEXT` type name between `` ` ``?

Reply:
Done, but note that this comment won't be turned into docs, as the type is private.